app/main.py

```diff
@@ -2,0 +1,35 @@
+class Animal:
+    alive = []
+
+    def __init__(self, name: "str", health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = False
+        Animal.alive.append(self)
+
+    def __str__(self) -> str:
+        return (f"{{Name: {self.name}, "
+                f"Health: {self.health}, Hidden: {self.hidden}}}")
```

Comment:
You have used string interpolation instead of concatenation. This is a good practice as it makes your code more readable and efficient. Well done!